earth_enterprise/src/google/protobuf/SConscript

```diff
@@ -70,6 +70,18 @@ stub_files = [
   'substitute.cc'
 ]
 
+# Remove gtest from the list of required libraries. For some reason the LIBS
+# variable includes "gtest" here, which makes libgeprotobuf unnecessarily
+# depend on gtest-devel. This is a hack to fix that problem, but it's a fairly
+# safe hack.
+try:
+  local_libs = local_env['LIBS']
+  local_libs = [lib for lib in local_libs if lib is not "gtest"]
+  local_env['LIBS'] = local_libs
+except:
```

Comment:
Minor: You might want to catch a specific exception, if all you're doing is check whether `local_libs` is defined, or use an `if` statement. This implementation will prevent you from seeing any exception that may occur in that block.